tests/integration/test_vid_club.py

```diff
@@ -23,23 +29,20 @@ def json():
 
 
 @pytest.mark.init
-def test_create_club_class():
-    vc = VidClub()
-    assert vc
+def test_default_credential_param():
+    assert vc.credentials != None and type(vc.credentials) == dict
 
 
 @pytest.mark.init
-def test_default_credential_param():
-    vc = VidClub()
-    assert vc.credentials != None and type(vc.credentials) == dict
+def test_create_club_class():
+    assert vc
```

Comment:
I am not sure what this test is checking, because you did not specify the expected behavior or the condition you want to test for the object 'vc'. The test case should include assertions that compare the actual values of the object's properties or methods with the expected values. 
```suggestion
    assert isinstance(vc, VidClub)
```

Reply:
I removed this test